README.md

```diff
@@ -1,185 +1,191 @@
-  Shuffles the order of the proxy list from <https://www.sslproxies.org/> instead of going from newest to oldest (as listed on the website). Defaults to `rand=False`
-
-```python
-proxy = FreeProxy(rand=True).get()
-```
-
-- **`anonym` parameter**
-  Return only proxies marked as anonymous. Defaults to `anonym=False`
-
-```python
-proxy = FreeProxy(anonym=True).get()
-```
-
-- **`elite` parameter**
-  Return only proxies marked as 'elite proxy'. Defaults to `elite=False`.
-
-```python
-proxy = FreeProxy(elite=True).get()
-```
-
-Please note: elite proxies are always anonymous. If you set `elite=True`, you will also be eliminating any non-anonymous proxies.
-
-- **`google` parameter**
-  If `True` it will only return proxies marked as "google". If `False`, it will not return proxies marked as "google". Defaults to `google=None`, which returns all proxies.
-
-```python
-proxy = FreeProxy(google=True).get()
-```
-
-- **`https` parameter**
-  If `True` it will only return proxies marked as HTTPS. Defaults to `https=False` - i.e. HTTP proxy (for HTTP websites).
-
-  Please note: HTTPS proxies work for both HTTP and HTTPS websites.
-
-```python
-proxy = FreeProxy(https=True).get()
-```
-
-You can combine parameters:
-
-```python
-proxy = FreeProxy(country_id=['US', 'BR'], timeout=0.3, rand=True).get()
-```
-
-If there are no working proxies with the provided parameters, the script will raise `FreeProxyException` with the message `There are no working proxies at this time.`.
-
-## CHANGELOG
-
----
-
-## [1.1.1] - 2023-02-18
-
-- Fixed https parameter error
-- Fixed additional loop issue when no proxy was found
-
-## [1.1.0] - 2022-11-12
-
-- Added new website to get proxies from <https://free-proxy-list.net>
-- Added new website to get proxies from <https://free-proxy-list.net/uk-proxy.html>
-- Added new website to get proxies from <https://www.us-proxy.org/>
-- Change lxml version to 4.9.1
-
-## [1.0.6] - 2022-01-23
-
-- Added `google` parameter
-- Added `https` parameter
-
-## [1.0.5] - 2022-01-07
-
-- Added `elite` parameter
-- Add exception class and raise exception instead of system exit
-- Change lxml version to 4.6.5
-
-## [1.0.4] - 2021-11-13
-
-- Fix proxy list default length
-
-## [1.0.3] - 2021-08-18
-
-- Change XPatch due to SSL proxies page update
-- Change lxml version
-
-## [1.0.2] - 2020-09-03
-
-- Added `anonym` parameter
-
-## [1.0.1] - 2020-03-19
-
-- Fix typos in readme
-- Fix urrlib3 exception `urllib3.exceptions.ProxySchemeUnknown: Not supported proxy scheme None`,
-- Fix imports
-
-## [1.0.0] - 2019-02-04
-
-- Initial release
-
-## License
-
----
-
-[MIT](https://github.com/jundymek/free-proxy/blob/master/LICENSE)
-
-**Free Software!**
+# Free-proxy
+
+![Version 1.1.1](https://img.shields.io/badge/Version-1.1.1-blue.svg)
+
+## Get free working proxies from <https://www.sslproxies.org/>, <https://www.us-proxy.org/>, <https://free-proxy-list.net/uk-proxy.html> and <https://free-proxy-list.net> and use them in your script
```

Comment:
why this and next strings in diff, while they are completely the same? something wrong with string ends?

Reply:
I updated the version (line 3) and lines 133-137 in README.md. It’s strange, but I can’t get GitHub to show only the updated lines.

> why this and next strings in diff, while they are completely the same? something wrong with string ends?